tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es3",
+    "target": "es5",
```

Comment:
What was the reason for this change? I think I set it to es3 because for some reason that resulted in a slightly smaller output size

Reply:
the `recoveryKey` getter, to allow it to generate `Object.defineProperty` with an accessor descriptor

Reply:
(we discussed it in https://ably-real-time.slack.com/archives/C02P7F81DPS/p1669125337243929 - you said this branch still had a smaller output size than main)

Reply:
ah i remember now, thank you 👍